src/containers/Member.js

```diff
@@ -213,6 +213,7 @@ MemberPage.propTypes = {
       key: PropTypes.string.isRequired,
       name: PropTypes.string.isRequired,
       name_en: PropTypes.string,
+      weight: PropTypes.number,
```

Comment:
Question: does fe need `weight` info ?
( current be code do not return `weight` value in get user route )

Reply:
fixed